server/src/permissions/constants.ts

```diff
@@ -0,0 +1,11 @@
+export enum Role {
+  admin = 'ADMIN',
+  owner = 'OWNER',
+  revoked = 'REVOKED',
+}
+
+export enum ApprovalStatus {
+  approved = 'APPROVED',
+  pending = 'PENDING',
+  denied = 'DENIED',
+}
```

Comment:
There are a lot of changes showing up that aren't part of your PR. Let's pair to rebase your branch to `main`